database-commons/src/main/java/io/cdap/plugin/db/batch/config/AbstractDBSpecificSinkConfig.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.db.batch.config;
+
+import com.google.common.collect.Maps;
+import io.cdap.cdap.api.annotation.Description;
+import io.cdap.cdap.api.annotation.Macro;
+import io.cdap.cdap.api.annotation.Name;
+import io.cdap.cdap.api.plugin.PluginConfig;
+import io.cdap.plugin.common.Constants;
+import io.cdap.plugin.db.ConnectionConfig;
+import io.cdap.plugin.db.connector.AbstractDBSpecificConnectorConfig;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+
+/**
+ * Abstract Config for DB Specific Sink plugin
+ */
+public abstract class AbstractDBSpecificSinkConfig extends PluginConfig implements DatabaseSinkConfig {
+  public static final String TABLE_NAME = "tableName";
+  public static final String TRANSACTION_ISOLATION_LEVEL = "transactionIsolationLevel";
```

Comment:
where is this variable used?

Reply:
Its being used in CloudSql MySql and Cloud PostgreSql sink can remove it for now as they don't use connections for now and add it in future PR.

Reply:
If we will add it back later, let's just keep it there.